src/components/Form/PhotoUpload.vue

```diff
@@ -102,7 +103,7 @@ import VueCropper from 'vue-cropperjs'
 
 export default {
   name: 'PhotoUpload',
-  props: ['color', 'default-image', 'multiple'],
+  props: ['color', 'default-image', 'multiple', 'default-text'],
```

Comment:
maybe we should replace "default-text" by a parameter like "upload-type" which can take the values "Photo", "Logo", "Header Image"

In the translation files we yould have

```
"Select",
"Drag&Drop",
"Foto",
"Logo",
"Header Image",
```

Reply:
i agree this comments => upload-type sounds more precise, i'll take the changes

Reply:
the logic for upload-type is set up